### PR TITLE
Simplify GitHub issue template

### PR DESCRIPTION
**Pull request by revolter**
- Replaced top quote with HTML code in case the user doesn't remove it
- Removed top instruction removal request as it's not needed anymore
- Replaced `this` with `these` so its clearer that the example on the second line must be removed too

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
 ## What happened instead?
 
-ℹ Please replace this with of what happened instead.  
+ℹ Please replace these with of what happened instead.  
 e.g. Pod A is missing the subspec B for target C.
 
 ## CocoaPods Environment
 
-ℹ Please replace this with the output of `pod env`.
+ℹ Please replace these with the output of `pod env`.
```
Comment (amorde) [inline .github/ISSUE_TEMPLATE.md]: This doesn't seem to be more clear than using `this` here - any particular reason for this change?

Comment (amorde) [reply .github/ISSUE_TEMPLATE.md]: (note: I did see your note in the PR description)

Comment (dnkoutso) [reply .github/ISSUE_TEMPLATE.md]: +1 why change these?

**Comment by dnkoutso** [comment]
Good with me if we revert back to using "this"

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (paulb777) [reply .github/ISSUE_TEMPLATE.md]: Perhaps "these two lines" to resolve the ambiguity of how much exactly should be replaced?

Comment (dnkoutso) [reply .github/ISSUE_TEMPLATE.md]: yeap also "these two lines" is good also.

Comment (amorde) [reply .github/ISSUE_TEMPLATE.md]: I agree either "these lines" or "these two lines" would be a worthwhile change

Comment (revolter) [reply .github/ISSUE_TEMPLATE.md]: "This" is too ambiguous (this line? this section?), and I think that "these two lines" is the clearer version for anybody, even non-native english speakers.

**Comment by revolter** [comment]
@dnkoutso, Let me know if the latest version is good 😄

**Comment by revolter** [comment]
Thank you for the merge 😃

**Pull request merged**